Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml

```diff
@@ -93,7 +94,7 @@ script:
     description: Retrieves a limited number of the indicators.
     execution: false
     name: unit42-get-indicators
-  dockerimage: demisto/taxii2:1.0.0.9904
+  dockerimage: demisto/taxii2:1.0.0.10275
```

Comment:
please revert this as it causes an 400 error, same issue as in mitre feed.

Reply:
Reverted